opencga-app/app/scripts/azure/arm/appgw/accesspolicies.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "vaultName": {
+            "type": "string"
+        },
+        "objectId": {
+            "type": "string",
+            "metadata": {
+                "description": "Specifies the object ID of a user, service principal or security group in the Azure Active Directory tenant for the vault. The object ID must be unique for the list of access policies. Get it by using Get-AzADUser or Get-AzADServicePrincipal cmdlets."
```

Comment:
Reference `az` commands, rather than PS cmdlets?

Reply:
Better explained in https://github.com/opencb/opencga/pull/1202/commits/438ce991d65fbb83ec0c1bc399c020be7eaa9b4